napi-inl.h

```diff
@@ -21,6 +21,12 @@
 #include <type_traits>
 #include <utility>
 
+#if defined(__clang__) || defined(__GNUC__)
+#define NAPI_NO_SANITIZE_VPTR __attribute__((no_sanitize("vptr")))
```

Comment:
I believe the macro def should be `NODE_ADDON_API_NO_SANITIZE_VPTR`

The `NAPI_` / `NODE_API_` prefix is meant for core.